picard/acoustid/__init__.py

```diff
@@ -23,7 +23,10 @@
 from picard import config, log
 from picard.const import FPCALC_NAMES
 from picard.util import find_executable
-from picard.util.xml import XmlNode
+from picard.acoustid.json_helpers import parse_recording
+
+# Not an unused import. Don't remove it
```

Comment:
I did change the line in tagger. Just thought it was better to keep all the acoustid imports in one module.